make_common/bob_robotics.mk

```diff
@@ -12,11 +12,20 @@ CPP_STANDARD ?= c++14
 
 # Build flags
 CXXFLAGS += -std=$(CPP_STANDARD) -Wall -Wpedantic -Wextra -MMD -MP
+
+# Include the root BoB robotics folder
+CURRENT_DIR := $(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
```

Comment:
I THINK, based on a past Makefile that did this, you can do
```Makefile
CURRENT_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
```
which is _slightly_ nicer